.travis.yml

```diff
@@ -2,8 +2,6 @@ sudo: required
 
 language: node_js
 node_js:
-  - "6"
```

Comment:
Would love to keep compatibility down to v6. Why we should not?

Reply:
Because async/await is used

Reply:
I will check what changes are needed to keep supporting v6, I'm not so sure its only async / await but I'll experiment with it

Reply:
@kr1sp1n @StarpTech [Added support for node v6+](https://github.com/DaniGuardiola/node-vault/commit/1836216df2514801a6755b466e52b0d0194ccb34) ([travis support](https://github.com/kr1sp1n/node-vault/commit/bdbeff1e06ece732e04c0cb2f2db205a54a19819))